src/Draco.Fuzzer/Testing/ParserFuzzer.cs

```diff
@@ -0,0 +1,31 @@
+using Draco.Compiler.Internal.Syntax;
+using static Draco.Compiler.Internal.Syntax.ParseNode;
+using Draco.Fuzzer.Testing.Generators;
+
+namespace Draco.Fuzzer.Testing;
+
+internal sealed class ParserFuzzer : ComponentFuzzer
+{
+    private IInputGenerator<IEnumerable<Token>> generator;
+
+    public ParserFuzzer(IInputGenerator<IEnumerable<Token>> generator)
+    {
+        this.generator = generator;
+    }
+
+    public override void RunEpoch()
+    {
+        var input = this.generator.NextExpoch();
+        try
+        {
+            // We just care about the parsing into compilation unit part
```

Comment:
Yep, totally fair. We won't care about anything else until we explicitly want to support stuff like a REPL